Dockerfile

```diff
@@ -25,12 +25,14 @@ RUN go build -mod=vendor -ldflags "-X github.com/prebid/prebid-server/version.Ve
 FROM ubuntu:18.04 AS release
 LABEL maintainer="hans.hjort@xandr.com" 
 WORKDIR /usr/local/bin/
-COPY --from=build /app/prebid-server/prebid-server .
-COPY static static/
-COPY stored_requests/data stored_requests/data
+COPY --chmod=a+xr --from=build /app/prebid-server .
```

Comment:
Making Prebid Server binary and other static resources to be non-writable

Reply:
Looks like this was released as part of 20.10 but was somehow missed from release notes (See: https://github.com/moby/moby/issues/34819#issuecomment-749570216) and requires `DOCKER_BUILDKIT=1` for it to work correctly (See: https://github.com/moby/moby/issues/34819#issuecomment-765637751).

We will change this to use the `RUN` command for now as this is a relatively new feature and might break for a lot of other people. Thanks for calling it out!